ui/components/ExpansionPanels.js

```diff
@@ -0,0 +1,13 @@
+import { Accordion, AccordionSummary } from "@mui/material";
+import { withStyles } from "@mui/styles";
+
+export const ExpansionPanel = withStyles({
+  root: { border: "1px solid rgba(0,0,0,.125)" },
+  expanded: { margin: "auto" },
+})(Accordion);
+
+export const ExpansionPanelSummary = withStyles({
+  root: { borderBottom: "1px solid rgba(0,0,0,.125)" },

```

Comment:
Danger, Will Robinson!

@meshery/meshery-ui-maintainers, should we be avoiding `root` styling?

Reply:
Haha, someone watched _Lost in Space_ huh?

Coming to fhe question, I think based in how mui works this will be limited to the div that the component creates. But, will let others confirm...

Reply:
we are no longer using make styles and the styles added here in the withstyles can be added in the theme.js file

Reply:
came out that root styling was not the problem on later work. So its fine to use "root" styling.